www/src/stores/thc.ts

```diff
@@ -35,13 +36,27 @@ export const thc = derived(
   null
 );
 
+export const questsRootCID: Readable<string | null> = derived(
+  thc,
+  ($thc, set) => {
+    if ($thc) {
+      retry(async () => {
+        const cid = await $thc.getQuestsRootCID();
+        const hashBuffer = arrayify(cid);
+        const ipfsHash = base58.encode(hashBuffer);
```

Comment:
I would recommend a more future-proof version, that doesn't make assumptions on the version of the CID. You need to install the library `multiformats`

```
import { CID } from "multiformats"

const ipfsHash = CID.decode(hashBuffer).toV0().toString();
```

Reply:
Good idea, done.